### PR TITLE
Update-windows-versions

### PR DESCRIPTION
**Pull request by bmckercher123**
### Update windows versions

- For CE, 3.15, 3.16, 3.17
- Updates for OSS in separate PR

### CODE REVIEW

File: calico-enterprise/getting-started/install-on-clusters/windows-calico/kubernetes/requirements.mdx [hunk 1]
```diff
@@ -71,7 +71,7 @@ For operator-managed Linux {{prodname}} clusters, three Linux worker nodes are r
 - Windows versions:
 
   - Windows Server 1809 (build Build 17763.1432 or greater)
```
Comment (coutinhop) [inline calico-enterprise/getting-started/install-on-clusters/windows-calico/kubernetes/requirements.mdx]:
```suggestion
  - Windows Server 1809 (build 17763.1432 or later)
```

File: calico-enterprise_versioned_docs/version-3.15/getting-started/install-on-clusters/windows-calico/kubernetes/requirements.mdx [hunk 2]
```diff
@@ -71,7 +71,7 @@ For operator-managed Linux {{prodname}} clusters, three Linux worker nodes are r
 - Windows versions:
 
   - Windows Server 1809 (build Build 17763.1432 or greater)
```
Comment (coutinhop) [inline calico-enterprise_versioned_docs/version-3.15/getting-started/install-on-clusters/windows-calico/kubernetes/requirements.mdx]:
```suggestion
  - Windows Server 1809 (build 17763.1432 or later)
```

File: calico-enterprise_versioned_docs/version-3.16/getting-started/install-on-clusters/windows-calico/kubernetes/requirements.mdx [hunk 3]
```diff
@@ -71,7 +71,7 @@ For operator-managed Linux {{prodname}} clusters, three Linux worker nodes are r
 - Windows versions:
 
   - Windows Server 1809 (build Build 17763.1432 or greater)
```
Comment (coutinhop) [inline calico-enterprise_versioned_docs/version-3.16/getting-started/install-on-clusters/windows-calico/kubernetes/requirements.mdx]:
```suggestion
  - Windows Server 1809 (build 17763.1432 or later)
```

**Review by coutinhop — DISMISSED** [review]
LGTM. Let's do EE right away and OSS at a later time. I added a couple of suggestions which don't have to do with your changes but I feel would improve and standardize the presentation (instead of referring to greater and later builds). Thanks @bmckercher123!

File: calico-enterprise/getting-started/install-on-clusters/windows-calico/kubernetes/requirements.mdx (hunk 1 above)
Comment (bmckercher123) [reply calico-enterprise/getting-started/install-on-clusters/windows-calico/kubernetes/requirements.mdx]: Good catch!

**Pull request merged**